elixir/apps/web/lib/web/live/gateway_groups/new.ex

```diff
@@ -128,4 +109,35 @@ defmodule Web.GatewayGroups.New do
 
     {:noreply, socket}
   end
+
+  defp docker_command(secret) do
+    """
+    docker run -d \\
+      --name=firezone-gateway-0 \\
+      --restart=always \\
+      -v /dev/net/tun:/dev/net/tun \\
+      -e FZ_SECRET=#{secret} \\
+      us-east1-docker.pkg.dev/firezone/firezone/gateway:stable
```

Comment:
Just wanted to mention that I intentionally left these command "as is" for now to make this PR only about visual changes.  I'll update all of these commands (gateways and relays) later when making changes that were discussed earlier today.